Sources/TarkaUI/Components/Search Bar/SearchTextField.swift

```diff
@@ -36,10 +36,12 @@ struct SearchTextField: View {
   private func performSearch() {
     guard searchBarVM.needDelaySearch else { return }
     searchBarVM.onEditing(searchBarVM.searchItem.text)
+    searchBarVM.searchText = searchBarVM.searchItem.text
   }
   
   private func updateSearchText(_ value: String) {
     searchBarVM.onEditing(value)
```

Comment:
Do we need `onEditing` now that there's a published value?

Reply:
@arvindhsukumar User can pick there needs, also I don't want to break the existing implementation